tests/user_test.go

```diff
@@ -1,76 +1,125 @@
 package easypost_test
 
 import (
-	"encoding/json"
-
+	"fmt"
 	"github.com/EasyPost/easypost-go/v2"
-	"github.com/dnaeon/go-vcr/cassette"
+	"reflect"
+	"strings"
 )
 
-func cleanAPIKeys(data interface{}) {
-	switch data := data.(type) {
-	case map[string]interface{}:
-		for k, v := range data {
-			switch v := v.(type) {
-			case string:
-				if k == "key" {
-					data[k] = "XXX"
-				}
-			case []interface{}, map[string]interface{}:
-				cleanAPIKeys(v)
-			}
-		}
-	case []interface{}:
-		for i := range data {
-			switch v := data[i].(type) {
-			case []interface{}, map[string]interface{}:
-				cleanAPIKeys(v)
-			}
-		}
-	}
+func (c *ClientTests) TestUserCreate() {
```

Comment:
Please skip this test like we've done in the other suites and leave the same comment as to why we skipped them. 

Remove the checked in cassette for this test.